src/Greeting.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreeting } from './greetingSlice';
+
+function Greeting() {
+  const dispatch = useDispatch();
+  const greeting = useSelector((state) => state.greeting);
+  const status = useSelector((state) => state.greeting.status);
+  const error = useSelector((state) => state.greeting.error);
+
+  useEffect(() => {
+    if (status === 'idle') {
+      dispatch(fetchGreeting());
+    }
+  }, [status, dispatch]);
+
+  return (
+    <div>
+      {status === 'loading' && <div>Loading...</div>}
+      {status === 'succeeded' && <div>{greeting}</div>}
+      {status === 'failed' && (
+        <div>
+          Error:
+          {' '}
+          {error}
+        </div>
+      )}
+    </div>
```

Comment:
- [ ] Great job working on the functionality so far but I think it would be great if you fix the issue since when I try to run your app I can this error instead of the messages so please try to fix the issue here to let your messages be displayed 🔥💪🏻♻🙏🏻🙏🏻

![image](https://github.com/kazim110/hello-react-front-end/assets/58553711/f528237e-dcd3-45e8-9eae-72e27eb7b063)